apps/portals/src/_Core.scss

```diff
@@ -299,6 +307,9 @@ button.dropdown-toggle.nav-button:hover {
   color: #404b63;
   box-shadow: none;
   outline: none;
+  border-radius: 0; 
+  border-right: 0;
+  border-left: 0;
```

Comment:
Adjust nav menu button bottom border when menu is open -- 

main | feature
:----:|:----:
![main_border-radius-selected](https://user-images.githubusercontent.com/26949006/232621752-2d4e359e-f78f-4ac3-a9ab-2b6c35595307.png) | ![feature_border-radius-selected](https://user-images.githubusercontent.com/26949006/232621742-8a318048-947c-43db-a866-6c5414560b0f.png)